.github/workflows/release.yaml

```diff
@@ -28,7 +27,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Ensure release does not already exist
-        if: startsWith(github.ref, 'refs/heads/main') == false
+        if: startsWith(github.ref, 'refs/heads/main')
```

Comment:
This looks like it's inverting the condition?

Reply:
Yeah, this should only run on `main` as a release won't actually be cut on another branch.